src/createAnimatedComponent.tsx

```diff
@@ -643,7 +648,7 @@ export default function createAnimatedComponent(
             const sharedElementTransition =
               this.props.sharedTransitionStyle ?? new SharedTransition();
             sharedElementTransition.registerTransition(
```

Comment:
We shouldn't even try to register SharedElementTransition on web

Reply:
I've changed the if condition ([3dec3e8](https://github.com/software-mansion/react-native-reanimated/pull/4866/commits/3dec3e89302ad5aa1bba9382f020d515ed95b6b5)), so it shouldn't be done on web now. 

When it comes to `findNodeHandle`, I'm aware that we use it in many other places, however I was focused on finding usages that are causing problems on web. I think that getting rid of this function should be discussed on a meeting, since it affects not only web version of reanimated, but also android and iOS.